autofit/graphical/messages/abstract.py

```diff
@@ -377,11 +398,11 @@ def check_valid(self) -> np.ndarray:
         return self.check_finite() & self.check_support()
 
     @property
-    def is_valid(self) -> bool:
+    def is_valid(self) -> np.bool_:
```

Comment:
I'd probably set the return type to bool unless there's some reason that doesn't work?

Reply:
mypy complained if it was `bool`